src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -824,6 +824,8 @@ settings.set( "lua.autocomplete", true )
 settings.set( "list.show_hidden", false )
 if term.isColour() then
```

Comment:
This check no longer matters in relation to the colours, since greyscale terminals can now have their palette set to any greyscale colour, meaning you might still want to set your directory colour to green, since that could be set to a shade of grey.